docs/unified-test-documentation/dasharo-compatibility/315-network-boot.md

```diff
@@ -1,13 +1,105 @@
 # Dasharo Compatibility: Network Boot
 
-## Test cases
+## PXE001.001 Dasharo Network Boot is available
 
-### PXE001.001 iPXE network boot
+**Test description**
+
+This test aims to verify that the `iPXE Network boot` option is available and
+whether you are entered to `Dasharo Network Boot Menu` after selecting it.
+
+**Test configuration data**
+
+1. `FIRMWARE` = Dasharo
+
+**Test setup**
+
+1. Proceed with the
+    [Generic test setup: firmware](../../generic-test-setup#firmware).
+
+**Test steps**
+
+1. Power on the DUT.
+1. Hold the `BOOT_MENU_KEY` to enter the UEFI Boot Menu.
+1. Select the `iPXE Network boot` option using the arrow keys and press `Enter`.
+
+**Expected result**
+
+1. The `iPXE Network boot` option is available.
+1. After selecting the `iPXE Network boot` option you should enter to
+    `Dasharo Network Boot Menu`.
+
+## PXE002.001 Dasharo network boot menu boot options order is correct
+
+**Test description**
+
+This test aims to verify that `Dasharo Network Boot Menu` contains all the
```

Comment:
In my opinion, this point should be written without direct reference to the user.

For example:

This test aims to verify that `Dasharo Network Boot Menu` contains all of the needed options which are in the correct order.

Reply:
applied